src/platform/forms/save-in-progress/SaveInProgressErrorPage.jsx

```diff
@@ -103,6 +107,17 @@ class SaveInProgressErrorPage extends React.Component {
           </div>
         );
         break;
+      case LOAD_STATUSES.forbidden:
+        content = (
+          <div>
+            <div className="usa-alert usa-alert-error background-color-only">
```

Comment:
Can we add a screenshot of this to the PR for content review? Also these errors should follow the [error message style guide](https://design.va.gov/patterns/messaging-error-messages). I think this will read like 

>We can't process this request. We're sorry. We can't give you access to this information. For help, please call the VA.gov help desk at 855-574-7286 (TTY: 711). We’re here Monday–Friday, 8:00 a.m.–8:00 p.m. ET.

Reply:
thanks, @rianfowler Yes, I've asked Dan to post screenshot to make sure this message is rendered correctly.

Reply:
![image](https://user-images.githubusercontent.com/6521723/82089451-82b82600-96a8-11ea-8777-fbd29486d85e.png)

Reply:
thanks, @Clark-W-Griswold for showing me how the 401 and 404 errors look. 

To align this 404 error with how the other look, please remove this part of the message: We can't process this request. So message will read: 
We're sorry. We can't give you access to this information. For help, please call the VA help desk at 855-574-7286 (TTY: 711). We're here Monday-Friday, 8:00 a.m.-8:00 p.m. ET. (edited)